.github/workflows/release.yaml

```diff
@@ -0,0 +1,66 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  actions: write
+  contents: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      # get the version of the current git tag
+      - name: Get tag version
+        id: vars
+        run: echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+      - name: Generate release
+        uses: 'marvinpinto/action-automatic-releases@v1.2.1'
+        with:
+          repo_token:
+            '${{ secrets.GITHUB_TOKEN }}'
+            # generate release name with the tag name that was just pushed
+          automatic_release_tag: ${{ steps.vars.outputs.tag }}
+          prerelease: false
+  Docker-Build:
```

Comment:
Update this to match what we do in `ci.yaml`, which I've recently changed to make the webhook work.  We need to have it update production vs. staging.  I'll have to do a bunch of work there to mimic the same setup I've done on staging.

For how, you could have it deploy to staging as well so we can test everything.